test/tests/functional/pbs_node_rampdown.py

```diff
@@ -55,6 +55,10 @@ def convert_time(fmt, tm, fixdate=False):
     return rv
 
 
+def cmp(a, b):
+    return (a > b) - (a < b)
+
+
```

Comment:
where are we using this function?

Reply:
> The cmp() function should be treated as gone, and the __cmp__() special method is no longer supported. Use __lt__() for sorting, __eq__() with __hash__(), and other rich comparisons as needed. (If you really need the cmp() functionality, you could use the expression (a > b) - (a < b) as the equivalent for cmp(a, b).)

https://docs.python.org/3.0/whatsnew/3.0.html#ordering-comparisons

Reply:
i am aware that cmp() is not there in PY3, I see only one use of the function you have defined in the file.
shall we use this logic directly over there